developer-docs-site/static/move_demo/sources/ACLMessageBoard.move

```diff
@@ -0,0 +1,98 @@
+module Examples::Messageboard {
+    use Std::Vector;
+    use Std::Signer;
+    use CoreFramework::Timestamp;
+    use Std::ACL::Self;
+    use Std::Event::{Self, EventHandle};
+
+    // Error map
+    const ACCOUNT_NOT_IN_ACL: u64 = 0;
+    const CANNOT_REMOVE_ADMIN_FROM_ACL: u64 = 1;
+
+
+    struct Messageboard has key, store {
```

Comment:
key is enough?

Reply:
try to understand better. Is this due to the messageboard is a top level resource and it is stored in global storage by default?